qa/1131

```diff
@@ -9,20 +9,25 @@ seq=`basename $0`
 echo "QA output created by $seq"
 
 . ./common.python
+. ./common.check
```

Comment:
common.python already brings in common.check so we don't need this new line AFAICT.